### PR TITLE
add pesistent volume metric

### PR DESCRIPTION
**Pull request by zouyee**
https://github.com/kubernetes/kube-state-metrics/issues/122
add pesistent volume metric

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/227)

### CODE REVIEW

**Comment by zouyee** [comment]
@brancz PTAL

File: collectors/persistentvolume.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	descPersistentVolumeStatusPhase = prometheus.NewDesc(
+		"kube_persistentvolume_status_phase",
+		"The phase indicates if a volume is available, bound to a claim, or released by a claim.",
+		[]string{
+			"namespace",
+			"persistentvolume",
```
Comment (andyxning) [inline collectors/persistentvolume.go]: `s/persistentvolume/name`

**Comment by andyxning** [comment]
one more nit: docs should be added also.

File: collectors/persistentvolume.go (hunk 1 above)
Comment (andyxning) [inline collectors/persistentvolume.go]: after looking at `kube_pod_status_phase`, seems using `persistentvolume` is fine. Sorry for the inconvenience.

**Pull request closed without merging**